libindy/src/services/pool/mod.rs

```diff
@@ -474,8 +459,8 @@ mod tests {
         }
 
         #[test]
-        fn pool_send_tx_works_for_closed_socket() {
-            test::cleanup_storage("pool_send_tx_works_for_closed_socket");
+        fn pool_send_tx_fails_for_closed_socket() {
```

Comment:
```suggestion
        fn pool_send_tx_works_for_closed_socket() {
```

Reply:
current name conversion for tests assumes "works" for non-happy path as well

Reply:
@AxelNennker if you will try to use github UI to "commit suggestion" do not forget manually line about DCO in commit message details